crates/ruff_python_semantic/src/definition.rs

```diff
@@ -33,14 +33,15 @@ pub enum ModuleKind {
 }
 
 /// A Python module.
-#[derive(Debug)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
Nit: Unrelated to this PR. It's a bit surprising that `ModuleSource` is defined in `visibility` when it is used for much more.

Reply:
Good call, I moved it.